docs/intents.rst

```diff
@@ -84,22 +84,22 @@ A privileged intent is one that requires you to go to the developer portal and m
     through code as well.
 
 Do I need privileged intents?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Does this need one more ~ ?
```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Sorry the question mark weirded me out a bit